packages/babel-helpers/src/helpers/usingCtx.js

```diff
@@ -0,0 +1,76 @@
+/* @minVersion 7.23.0 */
+
+function dispose_SuppressedError(suppressed, error) {
+  if (typeof SuppressedError !== "undefined") {
+    // eslint-disable-next-line no-undef
+    dispose_SuppressedError = SuppressedError;
+  } else {
+    dispose_SuppressedError = function SuppressedError(suppressed, error) {
+      this.suppressed = suppressed;
+      this.error = error;
+      this.stack = new Error().stack;
+    };
+    dispose_SuppressedError.prototype = Object.create(Error.prototype, {
+      constructor: {
+        value: dispose_SuppressedError,
+        writable: true,
+        configurable: true,
+      },
+    });
+  }
+  return new dispose_SuppressedError(suppressed, error);
+}
+
+export default function _usingCtx() {
+  var empty = {};
+  return {
+    s: [], // stack
+    e: empty, // error
+    using: function (value, isAwait) {
```

Comment:
I'm not sure that this PR is worth doing, but anyway we can improve this helper:
- use `u` and `d` rather than `using` and `dispose`
- `s` can be a local variable, rather than a property
- you might split `u` in `u` and `a` ("using" and "await using"), to save on the second parameter